src/Context/ContextProviderTrait.php

```diff
@@ -54,7 +54,7 @@ public function getProvidedContext($name) {
   public function getProvidedContextDefinition($name) {
     $definition = $this->getPluginDefinition();
     if (empty($definition['provides'][$name])) {
-      throw new ContextException(sprintf("The %s provided context is not valid.", $name));
```

Comment:
I think we should not touch the exceptions that are already fully namespaced. This patch should only replace the global exceptions that don't have a namespace.